docs/api/api-tokens.md

```diff
@@ -1,11 +1,13 @@
 ---
-title: API Tokens
+title: Using API Tokens
 ---
 
 <head>
-  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/reference-guides/about-the-api/api-tokens"/>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/api/api-tokens"/>
 </head>
 
+Rancher v2.8.0 introduced the [Rancher Kubernetes API](./api-reference.mdx) which can be used to manage Rancher resources through `kubectl` or the [web interface](./v3-rancher-api-guide.md#enable-view-in-api).
```

Comment:
The RK API is not accessible through the web interface referenced since the latter is the v3 API.

We also need to establish the (lack of) relationship between the token content on this page with the RK API and v3 API. Maybe some variation of the below (insert links where needed)?

```suggestion
Rancher v2.8.0 introduced the [Rancher Kubernetes API](./api-reference.mdx) which can be used to manage Rancher resources through `kubectl`. This page covers information on API tokens used with the Rancher CLI, kubeconfig files, Terraform, and the [v3 API browser](./v3-rancher-api-guide.md#enable-view-in-api).
```

Reply:
Updated with suggestions and added in links where I thought appropriate, thanks for reviewing Billy!